src/ProgressIndicator/ProgressStep.svelte

```diff
@@ -52,6 +52,7 @@
 </script>
 
 <!-- svelte-ignore a11y-mouse-events-have-key-events -->
+<!-- svelte-ignore a11y-role-supports-aria-props -->
 <li
   aria-disabled="{disabled}"
```

Comment:
I think we can safely remove this attribute. The Flagship repo doesn't use it,
either and it'll comply with the warning: https://github.com/carbon-design-system/carbon/blob/4bf871b5d2d34703d7b2b43fce0d7fa418fe298e/packages/react/src/components/ProgressIndicator/ProgressIndicator.js#L194

Reply:
Thanks for the thorough review – addressed in https://github.com/carbon-design-system/carbon-components-svelte/pull/1703/commits/66a5409e80d5f454c71966913b0db4a8913b1ada